### PR TITLE
[ACS-3548] heading added to resolve heading accessibility issue

### PR DESCRIPTION
**Pull request by jatin2008**
**Please check if the PR fulfills these requirements**

> - [x ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
heading was not present that gives Headings should not be empty issue

**What is the new behaviour?**
heading added to resolve heading accessibility issue

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by DenysVuika** [comment]
Please use the JIRA ticket for the pr title so that it gets wired with the JIRA

File: app/src/app/app.component.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<h1 class="sr-only">{{ pageHeading | translate }}</h1>
+<h1 class="sr-only" title="heading">{{ pageHeading | translate }}</h1>
```
Comment (DenysVuika) [inline app/src/app/app.component.html:1]: title should be localiseable

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: app/src/app/app.component.html (hunk 1 above)
Comment (jatin2008) [reply app/src/app/app.component.html]: @DenysVuika localization changes have been done now.

**Review by DenysVuika — APPROVED** [review]

**Comment by DenysVuika** [comment]
Can you update the PR title with the format "[JIRA-NUMBER] description"

**Pull request merged**